homeassistant/components/zwave_me/translations/en.json

```diff
@@ -0,0 +1,17 @@
+{
+    "config": {
+        "step": {
+            "user": {
+                "title": "ZWaveMe Creation",
+                "description": "Input you WebSocket URL and Token here: TODO Where get token",
+                "data": {
+                    "url": "URL",
+                    "token": "Token"
+                }
+            }
+        },
+        "abort": {
+            "single_instance_allowed": "Only a single instance is allowed."
```

Comment:
@balloob If we allow multiple instances, shall we change this too?

Reply:
Yes. We have a common string for already_configured that will be used by our unique_id api for config entries.

https://github.com/home-assistant/core/blob/9f15e7dcf4c0009954874a7fe35abb5a11c71094/homeassistant/strings.json#L63